app/models/detector/suggested_resource.rb

```diff
@@ -53,5 +53,27 @@ def calculate_fingerprint(old_phrase)
       # Rejoin tokens
       tokens.join(' ')
     end
+
+    # This replaces all current Detector::SuggestedResource records with a new set from an imported CSV.
+    #
+    # @note This method is called by the suggested_resource:reload rake task.
+    #
+    # @param input [CSV::Table] An imported CSV file containing all Suggested Resource records. The CSV file must have
+    #                           at least three headers, named "Title", "URL", and "Phrase".
+    def self.bulk_replace(input)
+      raise ArgumentError.new, 'Tabular CSV is required' unless input.instance_of?(CSV::Table)
+
+      # Need to check what columns exist in input
+      required_headers = %w[Title URL Phrase]
+      missing_headers = required_headers - input.headers
```

Comment:
It seems like the inputs are case-sensitive, in which case I'd suggest we underscore that in the method documentation above. (The exact values are already listed, but it may be helpful to explicitly note case sensitivity.)

Reply:
Yeah, good point - case sensitivity did bite me initially with this (the File path silently downcases the header values, while URI doesn't (among other changes). I'll add this to the docs.